components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -69,6 +71,7 @@
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/common/chrome_paths.h"
+#include "components/grit/brave_components_strings.h"
```

Comment:
is there a dep for this? I believe brave_ads/browser has check_includes = false

Reply:
Added.